.ci/system-tests/system_test.py

```diff
@@ -0,0 +1,769 @@
+# system_test.py - implements a group of tests that run appraisals on a client and server
+
+# TODO: test_01-test_11 will need to be implemented when the flamethrower 
```

Comment:
Not sure if we want to have references to the other project by name. Perhaps something to the effect of, "when the additional HIRS projects are rolled out to the new GitHub repo".